python/lsst/sconsUtils/tests.py

```diff
@@ -293,3 +305,38 @@ def junitPrefix(self):
             prefix += ".{0}".format(os.environ[controlVar])
 
         return prefix
+
+    def _pytestCoverage(self, prefix="pytest"):
```

Comment:
I'd rename this method `_getPytestCoverageCommand` or similar, to clarify what it does.

It should also have a `Returns` block.